client/app/hosting/task/hosting-task.controller.js

```diff
@@ -28,17 +29,21 @@ angular.module('App').controller(
     }
 
     loadPaginated({ pageSize, offset }) {
-      return this.Hosting.getTasksList(
-        this.$stateParams.productId,
-        pageSize,
-        offset - 1,
-      )
-        .then((tasks) => {
-          this.tasksList = tasks;
+      return this.$q.all({
+        hosting: this.Hosting.getHosting(this.$stateParams.productId),
+        tasks: this.Hosting.getTasksList(
+          this.$stateParams.productId,
+          pageSize,
+          offset - 1,
+        ),
+      })
+        .then((resp) => {
```

Comment:
You can use the destructuring parameters syntax:
```suggestion
        .then(({ hosting, tasks }) => {
```